### PR TITLE
chore: format versions.ts file using repo config

### PR DESCRIPTION
**Pull request by yannbf**
Issue: N/A

## What I did

When running `sb-generate-sb-packages-versions.js` the file will formatted with the same config as any other file in the monorepo has. This will prevent annoying merge conflicts or extra git changes because of auto-formatting from the editor.

## How to test

- [ ] Is this testable with Jest or Chromatic screenshots?
- [ ] Does this need a new example in the kitchen sink apps?
- [ ] Does this need an update to the documentation?

If your answer is yes to any of these, please make sure to include it in your PR.

### CODE REVIEW

File: lib/cli/scripts/generate-sb-packages-versions.js [hunk 1]
```diff
@@ -40,12 +41,19 @@ const run = async () => {
...
-    dedent`
+    prettier.format(
+      dedent`
       // auto generated file, do not edit
       export default ${JSON.stringify(packageToVersionMap, null, 2)}
-    `
+    `,
+      { ...prettierConfig, parser: 'typescript' }
+    )
   );
```
Comment (ndelangen) [inline lib/cli/scripts/generate-sb-packages-versions.js:57]: alternatively we could run `eslint <file-path> --fix`

I'd also propose we move the script to the root of the monorepo.

**Review by ndelangen — APPROVED** [review]
nice improvement, please update the lockfile. If you could apply my comments, that'd be great, but I understand if that's adding a bunch of work you're not ready to commit to, right now.

File: lib/cli/scripts/generate-sb-packages-versions.js (hunk 1 above)
Comment (yannbf) [reply lib/cli/scripts/generate-sb-packages-versions.js]: Done!

File: lib/cli/scripts/generate-sb-packages-versions.js [hunk 2]
```diff
@@ -47,6 +48,10 @@ const run = async () => {
       export default ${JSON.stringify(packageToVersionMap, null, 2)}
     `
   );
+
+  exec(`yarn eslint --fix ${path.join(__dirname, '..', 'src', 'versions.ts')}`, {
```
Comment (ndelangen) [inline lib/cli/scripts/generate-sb-packages-versions.js:52]:
```suggestion
  exec(`yarn lint:js:cmd --fix ${path.join(__dirname, '..', 'src', 'versions.ts')}`, {
```

File: lib/cli/package.json [hunk 3]
```diff
@@ -82,6 +82,7 @@
     "@types/semver": "^7.3.4",
     "@types/shelljs": "^0.8.7",
     "@types/update-notifier": "^5.0.0",
+    "prettier": ">=2.2.1 <=2.3.0",
```
Comment (ndelangen) [inline lib/cli/package.json:85]: remove?

**Review by ndelangen — APPROVED** [review]

**Pull request merged**